framework/db/mssql/QueryBuilder.php

```
@@ -129,52 +131,81 @@ public function renameColumn($table, $oldName, $newName)
-                    $defaultValue instanceof Expression ? $defaultValue : new Expression($defaultValue)
+                    $defaultValue instanceof Expression ? $defaultValue : new Expression($defaultValue),
                 );
             }
 
             $checkValue = $type->getCheckValue();
+
             if ($checkValue !== null) {
-                $sqlAfter[] = "ALTER TABLE {$tableName} ADD CONSTRAINT " .
-                    $this->db->quoteColumnName("CK_{$constraintBase}") .
-                    ' CHECK (' . ($defaultValue instanceof Expression ?  $checkValue : new Expression($checkValue)) . ')';
+                $columnCheckConstraint = $this->db->quoteColumnName("CK_{$constraintBase}");
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $columnCheckConstraint. Keep variable name length under 20. (undefined)

(LongVariable)